### ISSUE TITLE
Zimbra citation style does not get highlighted

### ISSUE THREAD

**Issue by antoine-levitt** [djcb/mu#937]
This is a feature request. Some mail clients (most notably Zimbra) cite as

Blabla

----- Original message -----

Blibli

Mu4e doesn't detect it as a citation, and therefore Blibli doesn't get highlighted. Gnus does this pretty well, maybe steal some code from there?

**Comment by djcb** [comment]
Can you please attach a (raw) example message? Thanks.

**Comment by antoine-levitt** [comment]
Well actually, most messages I get are in French. Looking in my inbox, here's a sample of what I've got, with user-agent

Replies :
-------- Message d'origine --------
User-Agent: Zimbra 8.0.6_GA_5922 (MobileSync - SAMSUNG-GT-I8730/100.40102)

----- Mail original -----
User-Agent: Zimbra 8.0.6_GA_5922 (ZimbraWebClient - SAF7 (Mac)/8.0.6_GA_5922)


forward:
----- Mail transféré -----
User-Agent: Zimbra 8.0.6_GA_5922 (ZimbraWebClient - FF49 (Mac)/8.0.6_GA_5922)

-------- Forwarded Message --------
User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:45.0) Gecko/20100101 Thunderbird/45.3.0

---------- Forwarded message ----------
From my gmail (no user agent set)

So, in summary, who knows. I guess anything surrounded by a number of dashes greater or equal to four means a new citation level?

Here's one full message, anonymized


From: someone <someone@something.com>
To: someone <someone@something.com>
Subject: RE : Re: Something
Date: Thu 03 Nov 2016 07:48:54 AM CET
User-Agent: Zimbra 8.0.6_GA_5922 (MobileSync - SAMSUNG-GT-I8730/100.40102)

Blbl

-------- Message d'origine --------
De : someone <someone@something.com>
Date : 03/11/2016  7:38  (GMT+01:00) 
A : someone <someone@something.com>
Objet : Blbl

Blbl

**Comment by Chris00** [comment]
MS office uses this style as well.

**Comment by djcb** [comment]
mu4e has an (experimental for now) `gnus` mode, ie. `(setq mu4e-view-use-gnus t)` that should take care of this.

**Comment by antoine-levitt** [comment]
That has a number of features missing, and does not use mu4e's customizations. Does this mean mu4e's article view will not be actively developed in favor of gnus integration?
